packages/aws-cdk/lib/commands/init/template-registry.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Registry of public CDK templates maintained by the CDK team.
+ *
+ * Teams can contact the CDK team to add their templates to this registry.
+ */
+
+export interface PublicTemplateInfo {
+  /** Name of the template source (repository or package) */
```

Comment:
Format here should be like:

/**
 * Name of the template source (repository or package)
 */